src/interfaces/Product.ts

```diff
@@ -0,0 +1,18 @@
+interface Product {
+    id: number;
+    title: string;
+    price: number;
+    description: string;
+    images: string[];
+    creationAt: Date;
+    updatedAt: Date;
+    category: {
+        id: number;
+        name: string;
+        image: string;
+        creationAt: Date;
+        updatedAt: Date;
+    };
+}
+
+export default Product;
```

Comment:
do not recommend u use default exports in your project at all, in smaller project ity is aceptable, but in a bigger one, u will get a mess if you will use default exportsd
https://dev.to/phuocng/avoid-using-default-exports-a1c#:~:text=Default%20exports%20can%20be%20imported,and%20using%20different%20import%20names.

Reply:
Fixed